templates/vue/src/components/Toolbar/index.vue

```diff
@@ -1,18 +1,27 @@
 <template>
   <div class="toolbar">
-    <tapestry-filter v-if="!showMap" style="z-index: 10;" />
-    <div v-show="canEdit || (!showMap && hasDepth)" class="slider-wrapper">
-      <help-button v-if="canEdit" />
-      <review-notifications v-if="canEdit && settings.submitNodesEnabled" />
-      <settings-modal-button
-        v-if="canEdit"
-        :max-depth="maxDepth"
-      ></settings-modal-button>
-      <tapestry-depth-slider
-        v-show="!showMap && hasDepth"
-        @change="updateViewBox"
-        @change:max-depth="maxDepth = $event"
-      ></tapestry-depth-slider>
+    <tapestry-filter v-if="!showMap" style="z-index: 10" />
+    <div
+      v-show="isLoggedIn"
```

Comment:
This erroneously hides the depth slider for non-logged in users.

Reply:
Fixed